CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## Unversioned
 
-- Major: Added support for Twitch's Chat Replies. [Wiki Page](https://wiki.chatterino.com/Features/#message-replies) (#3722, #3989, #4041)
+- Major: Added support for Twitch's Chat Replies. [Wiki Page](https://wiki.chatterino.com/Features/#message-replies) (
+  #3722, #3989, #4041, #4047)
```

Comment:
```suggestion
- Major: Added support for Twitch's Chat Replies. [Wiki Page](https://wiki.chatterino.com/Features/#message-replies) (#3722, #3989, #4041, #4047)
```

Reply:
Oh that's because I have auto formatting turned on. But this will be resolved by [your previous suggestion](https://github.com/Chatterino/chatterino2/pull/4047#discussion_r989552531) anyways.